src/udev/udev-builtin-net_id.c

```diff
@@ -765,7 +773,9 @@ static int names_mac(sd_device *dev, struct netnames *names) {
         if (r < 0)
                 return r;
 
-        i = strtoul(s, NULL, 0);
+        r = safe_atolu(s, &i);
+        if (r < 0)
```

Comment:
errors here and below were ignored previously, is it ok to return or will it break something? /cc @yuwata

Reply:
I think it is ok to return negative errno here. The caller of `names_mac()` will be ignored any errors.